pkg/controller/backup_configuration.go

```diff
@@ -0,0 +1,249 @@
+package controller
+
+import (
+	"fmt"
+
+	workload_api "github.com/appscode/kubernetes-webhook-util/apis/workload/v1"
+	batch_util "github.com/appscode/kutil/batch/v1beta1"
+	core_util "github.com/appscode/kutil/core/v1"
+	"github.com/appscode/kutil/tools/queue"
+	"github.com/appscode/stash/apis"
+	api_v1beta1 "github.com/appscode/stash/apis/stash/v1beta1"
+	"github.com/appscode/stash/client/clientset/versioned/scheme"
+	util2 "github.com/appscode/stash/client/clientset/versioned/typed/stash/v1beta1/util"
+	"github.com/appscode/stash/pkg/docker"
+	"github.com/appscode/stash/pkg/util"
+	"github.com/golang/glog"
+	"k8s.io/api/batch/v1beta1"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/client-go/tools/reference"
+)
+
+func (c *StashController) initBackupConfigurationWatcher() {
+	c.bcInformer = c.stashInformerFactory.Stash().V1beta1().BackupConfigurations().Informer()
+	c.bcQueue = queue.New("BackupConfiguration", c.MaxNumRequeues, c.NumThreads, c.runBackupConfigurationInjector)
```

Comment:
Use `api_v1beta1.ResourceKindBackupConfiguration` instead of `"BackupConfiguration"`

Reply:
resolved